src/redux/actions/index.test.js

```diff
@@ -0,0 +1,52 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './index';
+import * as types from '../constants/action-types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('actions', () => {
+  it('should change search option', () => {
+    const payload = 'title';
+    const expectedAction = {
+      type: types.CHANGE_SEARCH,
+      payload
+    };
+    expect(actions.changeSearch(payload)).toEqual(expectedAction);
+  });
+
+  it('should change sorting option', () => {
+    const payload = 'rating';
+    const expectedAction = {
+      type: types.CHANGE_SORTING,
+      payload
+    };
+    expect(actions.changeSorting(payload)).toEqual(expectedAction);
+  });
+
+  it('should trigger start search', () => {
```

Comment:
Description for this test case does not reflect reality. Tested action creator does not trigger anything, it's just creates action.